src/pages/manage-candidateship.vue

```diff
@@ -363,7 +363,7 @@ export default {
 
       let registeraction = {
         account: this.$dir.getAccount(this.$dir.ACCOUNT_CUSTODIAN),
-        name: 'nominatecane',
+        name: 'nominatecand',
```

Comment:
Shouldn't this be `nominatecane` now?

Reply:
Sorry, this got somehow lost in merging, I should have double-checked